src/CMakeLists.txt

```diff
@@ -1,31 +1,37 @@
 ADD_LIBRARY(redisai_obj OBJECT
-    redisai.c
-    backends.c
-    model.c
-    err.c
-    script.c
-    tensor.c
-    rmutil/alloc.c
-    rmutil/sds.c
-    rmutil/args.c
-    rmutil/heap.c
-    rmutil/priority_queue.c
-    rmutil/vector.c)
+        redisai.c
+        backends.c
+        model.c
+        err.c
+        script.c
+        tensor.c
+        rmutil/alloc.c
+        rmutil/sds.c
+        rmutil/args.c
+        rmutil/heap.c
+        rmutil/priority_queue.c
+        rmutil/vector.c)
 
-ADD_LIBRARY(redisai_tensorflow_obj OBJECT
-    backends/tensorflow.c
-    err.c
-    tensor.c)
+if(BUILD_TF)
+    ADD_LIBRARY(redisai_tensorflow_obj OBJECT
+            backends/tensorflow.c
+            err.c
+            tensor.c)
+endif(BUILD_TF)
```

Comment:
I'd skip that as endif(X) doesn't really add much over endif() in simple conditions. But these is mere aesthetics, you can keep it if you like it.